JobConfig/primary/RPC.fcl

```diff
@@ -17,8 +17,9 @@ physics.producers.generate : {
     ehi : 140.
     spectrumResolution : 0.1 #MeV
   }
-  pionDecayOff : true
+  pionDecayOff : true # will apply surv prob
   doHistograms : true
+  #SurvivalProbScaling : 1
```

Comment:
is this a new parameter that isn't implemented yet? Or are you taking the default value for survivalprobscaling?

Reply:
It was a possibility, but I dont think its really needed